products/mssqlserver.md

```diff
@@ -67,6 +72,24 @@ releases:
     eol: 2019-07-09
     extendedSupport: 2022-07-12
     latest: "10.50.6560.0"
+    link: https://support.microsoft.com/help/4057113
+
+-   releaseCycle: "2008"
+    releaseLabel: "2008 SP4"
+    releaseDate: 2008-11-06
+    support: 2014-07-08
+    eol: 2019-07-09
+    extendedSupport: 2022-07-12
+    latest: "10.0.6556.0"
+    link: https://support.microsoft.com/help/4057114
+
+-   releaseCycle: "2005"
+    releaseLabel: "2005 SP4"
+    releaseDate: 2006-01-14
+    support: 2011-04-12
+    eol: 2016-04-12
+    latest: "9.00.5324.0"
```

Comment:
Just to double check, how do we decide between `9.0.1399.06` and `9.0.5000` (listed under the SP4 column).

Source: https://sqlserverbuilds.blogspot.com/

<img width="462" alt="image" src="https://user-images.githubusercontent.com/584253/216058479-b5dbc558-8162-47b1-a419-af3c70de5b15.png">

Reply:
Version `9.0.1399.06` is for the RTM without any service pack installed.

I don't know this blog, I think it is not updated to the latest. I took `9.00.5324.0` from here: https://learn.microsoft.com/de-de/troubleshoot/sql/releases/download-and-install-latest-updates#sql-server-2005 what refers to https://support.microsoft.com/help/2716427 at the column `File version`.